cmd/bee/cmd/db.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ethersphere/bee/pkg/localstore"
+	"github.com/spf13/cobra"
+)
+
+func (c *command) initDBCmd() {
+	cmd := &cobra.Command{
+		Use:   "db",
+		Short: "Perform basic DB related operations",
+	}
+
+	dbExportCmd(cmd)
+	dbImportCmd(cmd)
+
+	c.root.AddCommand(cmd)
+}
+
+func dbExportCmd(cmd *cobra.Command) {
+	c := &cobra.Command{
+		Use:   "export <filename>",
+		Short: "Perform DB export to a file. Use \"-\" as filename in order to write to STDOUT",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			if (len(args)) != 1 {
+				return cmd.Help()
+			}
+			v, err := cmd.Flags().GetString(optionNameVerbosity)
+			if err != nil {
+				return fmt.Errorf("get verbosity: %v", err)
+			}
+			v = strings.ToLower(v)
+			logger, err := newLogger(cmd, v)
+			if err != nil {
+				return fmt.Errorf("new logger: %v", err)
+			}
+
+			dataDir, err := cmd.Flags().GetString(optionNameDataDir)
+			if err != nil {
+				return fmt.Errorf("get data-dir: %v", err)
+			}
+			if dataDir == "" {
+				return errors.New("no data-dir provided")
+			}
+
+			fmt.Printf("starting export process with data-dir at %s\n", dataDir)
```

Comment:
Maybe to use logger here instead the print? Mainly for having the timestamps here and on the line where the export ends. WDYT?